examples/tests/NavigationSampleTests.elm

```diff
@@ -0,0 +1,35 @@
+module NavigationSampleTests exposing (all)
+
+import Expect
+import NavigationSample exposing (program)
+import Test exposing (..)
+import Test.Html.Event as Event
+import Test.Html.Query exposing (..)
+import Test.Html.Selector exposing (..)
+import TestContext exposing (..)
+
+
+all : Test
+all =
+    describe "NavigationSample"
+        [ test "renders navigation history" <|
+            \() ->
+                NavigationSample.program
+                    |> start
+                    |> simulate (find [ class "bears" ]) Event.click
+                    |> simulate (find [ class "cats" ]) Event.click
+                    |> expectView
+                    |> find [ class "history" ]
+                    |> Expect.all
+                        [ has [ text "#bears" ]
```

Comment:
when this is run in real life, won't the text be the entire page URL, not simply the hash part?